components/src/dynamo/profiler/utils/model_info.py

```
@@ -123,6 +124,66 @@ class ModelInfo(BaseModel):
+        with open(config_path) as f:
+            return json.load(f)
+
+    try:
+        from aiconfigurator.sdk.utils import get_model_config_from_model_path
+
+        model_config = get_model_config_from_model_path(str(model_name_or_path))
+        raw_config = model_config.get("raw_config")
+        if raw_config:
+            return raw_config
+    except Exception:
+        pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)